pyodide_build/pywasmcross.py

```diff
@@ -262,6 +262,12 @@ def handle_command(line, args, dryrun=False):
         # Don't include any system directories
         if arg.startswith("-L/usr"):
             continue
+        for l in args.replace_libs.split(";"):
```

Comment:
Would you want to do this splitting at the beginning of the function instead of repeating it every iteration?

Reply:
doh, fixing now